hazelcast/include/hazelcast/client/reliable_topic.h

```diff
@@ -179,6 +179,7 @@ class HAZELCAST_API reliable_topic : public proxy::ProxyImpl
             }
 
             auto runner = this->shared_from_this();
+            std::shared_ptr<int> a;
```

Comment:
🤔

Reply:
this was mistakenly changed. Reverted back.